server-user-stories-intro.md

```diff
@@ -0,0 +1,43 @@
+# User Stories for the Solid Server
+
+To enable more people to get involved, to improve code quality to
+production grade and to accelerate development, I would like to hear
+your User Stories for the server.
+
+This is to better understand better what people want and need from the
+server. User Stories are short descriptions of a need written from the
+perspective of a user,
+[see e.g. this article](https://www.mountaingoatsoftware.com/agile/user-stories). They
```

Comment:
"see e.g. this article" would be better "see this article for an example"

Reply:
yeah, that's better!